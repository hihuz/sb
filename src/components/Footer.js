import React from "react";

function Footer({ content }) {
  return <footer dangerouslySetInnerHTML={{ __html: content }} />;
}

export default Footer;
