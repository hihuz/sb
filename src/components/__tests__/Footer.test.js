import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import Footer from "../Footer";

describe("Footer", () => {
  test("Snapshot", () => {
    const component = shallow(<Footer />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });

  test("Should display the text passed as content props", () => {
    const content = "Hello";
    const component = shallow(<Footer content={content} />);
    const expected = content;
    const actual = component.text();
    expect(actual).toEqual(expected);
  });
});
