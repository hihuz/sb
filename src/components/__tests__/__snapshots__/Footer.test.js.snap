// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot 1`] = `
<footer
  dangerouslySetInnerHTML={
    {
      "__html": undefined,
    }
  }
/>
`;
